roles/libvirt_manager/tasks/start_manage_vms.yml

```diff
@@ -177,7 +177,7 @@
       set -o pipefail;
       cat ~/.ssh/authorized_keys |
       ssh -v {{ _user }}@{{ vm_ip.nic.host }} "cat >> ~/.ssh/authorized_keys"
-  retries: 5
+  retries: 18
```

Comment:
why 18 and not 20? I know, sounds stupid question, but... why? :)

Reply:
:) total wait time is 180 seconds which would translate to 3 minutes.

Reply:
should be enough - maybe we can even set a parameter for the future?

Reply:
this patch did not work in the failing environment. Looks like the problem is elsewhere.